src/test/java/htmlpublisher/HtmlPublisherIntegrationTest.java

```diff
@@ -182,6 +182,40 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         assertFalse(new File(build.getRootDir(), "htmlreports/reportnameB/htmlpublisher-wrapper.html").exists());
     }
 
+    @Test
+    public void testUseWrapperFileDirectly() throws Exception {
+        FreeStyleProject p = j.createFreeStyleProject("use_wrapper_job");
+        final String reportDir = "autogen_use_wrapper";
+        p.getBuildersList().add(new TestBuilder() {
+            public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
+                    BuildListener listener) throws InterruptedException, IOException {
+                FilePath ws = build.getWorkspace().child(reportDir);
+                ws.child("tab1.html").write("hello", "UTF-8");
+                ws.child("tab2.html").write("hello", "UTF-8");
+                return true;
+            }
+        });
+
+
+        HtmlPublisherTarget target1 = new HtmlPublisherTarget("tab1", reportDir, "tab1.html", true, true, false);
+        //default behavior is to not use wrapper file directly
```

Comment:
That seems to contradict https://github.com/jenkinsci/htmlpublisher-plugin/blob/126319b8aefa14601d1893bfecf964f5d37c0b3a/src/main/resources/htmlpublisher/HtmlPublisherTarget/config.jelly#L41